joblib/test/test_hashing.py

```diff
@@ -93,8 +93,8 @@ def f(self, x):
 def test_trival_hash(obj1, obj2):
     """Smoke test hash on various types."""
     # Check that 2 objects have the same hash only if they are the same.
-    is_hash_equal = (hash(obj1) == hash(obj2))
-    assert is_hash_equal == (obj1 is obj2)
```

Comment:
I think this second paranthesis is needed. Not having them failed tests for me. Because probably `is_hash_equal == obj1 is obj2` will be true if all are not `None`.